auctions/templates/bid.html

```diff
@@ -1,9 +1,10 @@
 {# TODO: switching to jinja with 'set' would eliminate the extra <form> #}
 {% load staticfiles %}
+   <link rel="stylesheet" href="//cdn.jsdelivr.net/pure/0.5.0/pure-min.css">
```

Comment:
I'd prefer to load it this way from the template on the backend. That way we don't have to push an update to the extension if/when we want to update our pure-css version from 0.5.0 to the next version. I'm trending to as few static resources in the extension as possible.